packages/connect/src/router.ts

```diff
@@ -131,14 +138,42 @@ export function createConnectRouter(
       );
       return this;
     },
-    rpc(service, method, implementation, options) {
-      const { protocols } = whichProtocols(options, base);
-      handlers.push(
-        createUniversalMethodHandler(
-          createMethodImplSpec(service, method, implementation),
-          protocols,
-        ),
-      );
+    rpc(
+      serviceOrMethod: ServiceType | MethodType,
+      methodOrImpl: MethodInfo | MethodImpl<MethodInfo<AnyMessage>>,
```

Comment:
No need to specify the default type parameter:

```suggestion
      methodOrImpl: MethodInfo | MethodImpl<MethodInfo>,
```